src/index.jsx

```diff
@@ -4,6 +4,8 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import AdminDashboard from './pages/AdminDashboard/AdminDashboard';
+import UploadCSVTemp from './pages/UploadCSVTemp/UploadCSVTemp';
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.25.4/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing file extension for "./pages/UploadCSVTemp/UploadCSVTemp"